DQMOffline/Trigger/plugins/GENERICDQM.h

```diff
@@ -0,0 +1,45 @@
+#ifndef DQMOffline_Trigger_GENERICDQM_H
+#define DQMOffline_Trigger_GENERICDQM_H
+
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+struct MEbinning {
+  int nbins;
+  double xmin;
+  double xmax;
+};
+
+struct OBJME {
+  MonitorElement* numerator;
+  MonitorElement* denominator;
+  inline void clear(){
```

Comment:
Hi, I now wonder how it will affect various developments using this base class in 93x and backport to 92x. For example, #19561 and #19585 have DQM codes with the initializations. Are they okay with the current base class that uses the clear method?